chart/templates/mayastor/agents/core/agent-core-deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
             requests:
               cpu: {{ .Values.agents.core.resources.requests.cpu | quote }}
               memory: {{ .Values.agents.core.resources.requests.memory | quote }}
-          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/{{ .Chart.Name }}-agent-core:{{ .Values.image.tag }}"
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/{{ .Chart.Name }}-agent-core:{{ default .Values.image.tag .Values.image.tags.controlPlane }}"
```

Comment:
You need to update tags to repoTags in all references.

Reply:
I missed out on that . I will update the files shortly.

Reply:
Updated the deployment yamls